addons/mrp/models/mrp_unbuild.py

```diff
@@ -76,17 +77,12 @@ class MrpUnbuild(models.Model):
     @api.depends('company_id', 'product_id')
     def _compute_allowed_mo_ids(self):
         for unbuild in self:
-            if unbuild.product_id:
-                domain = [
-                    ('state', '=', 'done'),
-                    ('product_id', '=', unbuild.product_id.id),
-                    ('company_id', '=', unbuild.company_id.id)
-                ]
-            else:
-                domain = [
+            domain = [
                     ('state', 'in', ['done', 'cancel']),
```

Comment:
I don't get it, you want to accept 'cancel' MO or not. IMO, this line should be `('state', '=', 'done')`. What do you think about it ?

Reply:
My Bad.
Indeed, It should be ('state', '=', 'done').
PR updated.